src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -136,6 +145,8 @@ const BlueprintSearch = ({ blueprintsTotal }: blueprintSearchProps) => {
   const dispatch = useAppDispatch();
   const debouncedSearch = useCallback(
     debounce((filter) => {
+      dispatch(setBlueprintsOffset(0));
+      dispatch(imageBuilderApi.util.invalidateTags([{ type: 'Blueprints' }]));
```

Comment:
have you tested ivalidation is needed? with search we have different parameters, so it might not be needed to invalidate...
Might be a good idea anyway, so I'm not opposed to keeping it.